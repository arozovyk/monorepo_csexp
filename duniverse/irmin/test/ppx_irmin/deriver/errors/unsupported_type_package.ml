module type S = sig
  val x : unit
end

type t = (module S) [@@deriving irmin]
