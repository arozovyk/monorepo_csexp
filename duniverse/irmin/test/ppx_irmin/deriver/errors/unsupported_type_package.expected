File "unsupported_type_package.ml", line 5, characters 0-38:
Error: ppx_irmin: package type (module S) encountered. Package types are not Irmin-serialisable.
